### ISSUE TITLE
export BEM

### ISSUE THREAD

**Issue by awinogradov** [awinogradov/react-bl#24]
src/core/bem

maybe it should be another package for build bem-components in react without react-bl

**Comment by Guria** [comment]
Yeah, but I think it's too early. Once it will be stable it worth to be extracted to separate package.

**Comment by awinogradov** [comment]
You are right. It's issue for the future.
